assert_test.go

```diff
@@ -0,0 +1,23 @@
+package parlor_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/blocky/parlor"
+)
+
+type Fooer interface {
+	Foo()
+}
+
+type AFooer int
+
+func (AFooer) Foo() {}
+
+func TestTypePredicate(t *testing.T) {
+	assert.True(t, parlor.TypePredicate[Fooer](AFooer(0)))
```

Comment:
I think that `AFooer` being an `int` makes this test less readable. `myFeeling.strength = 2`

Reply:
updated